test/prescan_connector_test.dart

```diff
@@ -239,7 +239,7 @@ void main() {
                           failure: null)
                     ]));
 
-            completer.completeError(null);
+            completer.completeError("");
```

Comment:
This is a "no". The official Dart recommendation is to throw either exceptions or errors. Try `Exception()` or alike.